synapse/lib/nexus.py

```diff
@@ -589,10 +589,11 @@ async def _tellAhaReady(self, status):
 
         try:
             await self.cell.ahaclient.waitready(timeout=5)
-            ahainfo = await self.cell.ahaclient.getCellInfo()
+            proxy = await self.cell.ahaclient.proxy(timeout=5)
```

Comment:
This timeout should probably be user configurable? Probably the same config value as above.